tests/python_raise/raise.py

```diff
@@ -0,0 +1,16 @@
+raise
+raise ValueError
+raise ValueError("error")
+
+raise NotImplementedError(
+    'The SimpleListFilter.lookups() method must be overridden to '
+    'return a list of tuples (value, verbose value).'
+)
+
+raise forms.ValidationError(
+    self.error_messages['invalid_login'],
+    code='invalid_login',
+    params={
+        'username': self.username_field.verbose_name
+    }
+)
```

Comment:
Can you add a test of something like

```py
raise SomeErrorWithAReallyReallyVeryVeryExtremelyExtensivelyLongNameSoItBreaks()
```

IMO it should print as

```py
raise \
    SomeErrorWithAReallyReallyVeryVeryExtremelyExtensivelyLongNameSoItBreaks()
```

Reply:
Sure, I'll do, I've been thinking that maybe we should break stuff in parenthesis like:

```python
raise (
    SomeErrorWithAReallyReallyVeryVeryExtremelyExtensivelyLongNameSoItBreaks()
)
```

For now I'll keep `\` but maybe we can see if other people also like wrapping with `()`

Reply:
I like that better, but maybe that’s because we do it that way in JS.

Reply:
Yeah, we should probably agree on a general strategy for this. I've been using `\`, mainly because that's how I've coded in the past, but I think PEP8 prefers parens:
> The preferred way of wrapping long lines is by using Python's implied line continuation inside parentheses, brackets and braces. Long lines can be broken over multiple lines by wrapping expressions in parentheses. These should be used in preference to using a backslash for line continuation.
>
>Backslashes may still be appropriate at times. For example, long, multiple `with`-statements cannot use implicit continuation, so backslashes are acceptable:
>
>...
>
>Another such case is with `assert` statements.

It should be fairly straight-forward to find all the places we're currently using `\` by looking for usages of `escapedLine` and `escapedSoftline`.